src/anemoi/models/layers/attention.py

```diff
@@ -88,11 +130,23 @@ def forward(
         value = shard_heads(value, shapes=shapes, mgroup=model_comm_group)
         dropout_p = self.dropout_p if self.training else 0.0
 
-        if _FLASH_ATTENTION_AVAILABLE:
+        if self.use_flash_attention:
```

Comment:
Is this boolean persistant in the checkpoint?

Reply:
And also, why do we test it in `forward` when we used it in `__init__`?

Reply:
It should be persistent and it is also part of the model metadata.

We need it in the init to set the attention function and we need it in the forward because we perform slightly different operations for the two attention implementations.